indexx.js

```diff
@@ -10,8 +26,21 @@ let buttons = document.querySelectorAll('button');
             if (e.target.value == "=") {
                 entry = eval(entry);
                 document.querySelector("input").value = entry;
-
-            } else if (e.target.value == "C") {
+            }else if (e.target.id == "M+") {
+                st_memory=eval(st_memory+"+"+entry);
+                document.querySelector("input").value = st_memory;
```

Comment:
`document.querySelector('input')` is repeated too many times.
This can be stored in a variable at top and can be re-used.